src/redux/categories/categories.js

```diff
@@ -0,0 +1,21 @@
+const CHECK_STATUS = 'book-store-react/categories/CHECK_STATUS';
+
+// Reducer
+export const categoriesReducer = (state = [], action) => {
```

Comment:
- [ ] I kindly suggest that you export `categoriesReducer` as default because that is a project requirement.

## EXAMPLE
```
export default function ReactApp() {
  return <p>Exported Using Default Export</p>
}
```